engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -79,6 +78,13 @@ case class WorkflowExecutionActor(workflowDescriptor: EngineWorkflowDescriptor,
 
   when(WorkflowExecutionPendingState) {
     case Event(ExecuteWorkflowCommand, _) =>
+      // TODO WOM: Remove this when conditional and scatters are supported. It prevents the workflow from hanging
+      if(workflowDescriptor.namespace.innerGraph.nodes.collect({
```

Comment:
ToL `collectFirst` to stop at the first one, but hopefully this is not long for this world. 😄